### PR TITLE
Read book changes

### PR DESCRIPTION
**Pull request by alexanbj**
This pull request overhauls the book reading experience.

* Moves reading to it's own page.
* Drop the footer with navigation (that didn't work on mobile).
* Swipe to navigate chapters on touch enabled devices.
* Tap sides to navigate chapters on touch enabled devices.
* Tap between sides to enable navigation overlay on touch enabled devices 

Closes GlobalDigitalLibraryio/issues#104

### CODE REVIEW

File: components/Reader/Reader.js [hunk 1]
```diff
@@ -8,71 +8,234 @@
...
+            onRequestPrev={this.props.onRequestPrevious}
+            disableNext={disableNext}
+            disablePrev={disablePrev}
+          />
+          <Card>
+            <Toolbar
+              showOnMobile={this.state.showOverlay}
+              onRequestClose={this.props.onRequestClose}
+            />
+            <Box px={[40, 120]} pb={20} flex="1 0 auto">
```
Comment (kstigen) [inline components/Reader/Reader.js]: Mer et spørsmål enn en kommentar. Klarer man å holde oversikt over alle slike verdier, når de er hardkodet rundt omkring in applikasjonen?

File: components/Reader/Reader.js [hunk 2]
```diff
@@ -8,71 +8,234 @@
...
+        state => ({ chapter: state.chapter + 1 }),
+        this.changeChapterInUrl,
+      );
+    }
+  };
+
+  onRequestPrevious = () => {
+    if (this.state.chapter > 1) {
+      this.loadChapter(this.state.chapter - 1);
+      this.loadChapter(this.state.chapter - 2);
```
Comment (kstigen) [inline components/Reader/Reader.js]: Første kapittel har id 0?

File: components/Reader/Reader.js [hunk 3]
```diff
@@ -8,71 +8,234 @@
...
+  chapters: { [number]: Chapter },
+  chapter: number,
+};
+
+type ReaderContainerProps = {
+  book: Book,
+  initialChapter: ?string,
+};
+
+export default class ReaderContainer extends React.Component<
```
Comment (kstigen) [inline components/Reader/Reader.js]: Hadde kanskje vært fornuftig med noen tester på logikken i denne klassen?

File: components/Reader/Reader.js (hunk 2 above)
Comment (alexanbj) [reply components/Reader/Reader.js]: Yes, but "Chapter 0 / 12" doesn't look very pretty unless you're an engineer, so we subtract by one in the `loadChapter` method so we lookup correctly in the array there. In other words, we use one indexed chapters.

File: components/Reader/Reader.js (hunk 1 above)
Comment (alexanbj) [reply components/Reader/Reader.js]: På sikt ser jeg for oss at vi får til en slags spacing scale

File: components/Reader/Reader.js (hunk 3 above)
Comment (alexanbj) [reply components/Reader/Reader.js]: Tests have been added

**Pull request merged**